python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -300,18 +345,18 @@ def upsert_athena_table(
 
         dataset_schema_properties = SchemaMetadataClass(
             schemaName=metadata.name,
-            platform="urn:li:dataPlatform:athena",
-            version=metadata.major_version,
+            platform=metadata.platform.id,
+            version=1,
             hash="",
             platformSchema=OtherSchemaClass(rawSchema=""),
             fields=[
                 SchemaFieldClass(
-                    fieldPath=f"{column['name']}",
+                    fieldPath=f"{column.name}",
                     type=SchemaFieldDataTypeClass(
-                        type=DATAHUB_DATA_TYPE_MAPPING[column["type"]]
+                        type=DATAHUB_DATA_TYPE_MAPPING[column.type]  # type: ignore
```

Comment:
Note for later: we probably need to handle the case where `column.type` is missing from the dictionary